versions/3.1.0.md

```diff
@@ -1458,12 +1455,15 @@ Binary content transferred with base64 encoding:
 
 ```yaml
 content:
-    image/png:
+    text/plain:
```

Comment:
I believe this should stay as image/png to convey the semantics of the HTTP payload.

Reply:
@darrelmiller I don't remember why I did that, but I'll take your word for it.  I am having a difficult time finding a reference (which I was hoping would remind me why I thought this was correct).  But yeah, the JSON Schema provides all of the processing information and the logical output is `image/png` so that makes sense.  And of course the text that asserted this should indeed be removed.  Lemme do that right now...